cmd/lambda/main.go

```
@@ -47,3 +54,21 @@ func handler(config ArchiveToGoogleSheetsConfig) error {
 	)
 	return nil
 }
+
+func initSentry(dsn string) {
+	err := sentry.Init(sentry.ClientOptions{
+		Dsn:         dsn,
+		EnableLogs:  true,
+		Environment: getEnv("APP_ENV", "production"),
```

APP_ENV comes from STAGE (via `envName`), and is typically a short name like "prod", but you have "production" as the fallback. I even wonder whether a fallback is a good idea, or perhaps it should fail if not defined.